cmd/ctr/import.go

```diff
@@ -6,31 +6,63 @@ import (
 	"os"
 
 	"github.com/containerd/containerd"
+	dockerimporter "github.com/containerd/containerd/contrib/importer/docker"
+	"github.com/containerd/containerd/importer"
+	ociimporter "github.com/containerd/containerd/importer/oci"
 	"github.com/containerd/containerd/log"
 	"github.com/urfave/cli"
 )
 
 var imagesImportCommand = cli.Command{
-	Name:        "import",
-	Usage:       "import an image",
-	ArgsUsage:   "[flags] <ref> <in>",
-	Description: `Import an image from a tar stream.`,
+	Name:      "import",
+	Usage:     "import an image",
+	ArgsUsage: "[flags] <ref> <in>",
+	Description: `Import an image from a tar stream.
+Implemented formats:
+- oci.v1     (default)
+- docker.v1
+
+Selector string specification:
+- oci.v1:    containerd reference string (e.g. example.com/foo/bar:tag@digest) or reference object string (e.g. tag@digest).
+             Locator part in reference string is ignored.
+- docker.v1: Docker RepoTag string (e.g. example.com/foo/bar:tag)
+             Unlike containerd ref string, Docker RepoTag string MUST not contain digest. So don't put digest in selector.
+             Selector needs to contain locator part at the moment. (e.g. "docker.io/library/busybox:latest" is correct but "busybox:latest" not)
+
+The default selector string is <ref>, but it is not guaranteed to be a valid selector string, depending on the importer implementation.
+`,
 	Flags: []cli.Flag{
```

Comment:
This selector interface might be too much overloaded. RFC.

Reply:
Can you use the filter package?

Reply:
yes, I'll try